src/styles/_stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
There is an extra space between `&--4` and `:nth-child(-n + 4)`. According to the code style rules, there should not be a space between a block element and its modifier.